project_risk/menuitems.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why use a separate file instead of including this code in project/project_risk/views/project_risk_view.xml ?

Reply:
The previous version 11.0 locate the file here, now move the file to views directory.

Thanks